src/Symfony/Component/Dotenv/Dotenv.php

```diff
@@ -50,6 +51,33 @@ public function load(string $path, string ...$paths): void
         $this->doLoad(false, $path, $paths);
     }
 
+    /**
+     * Loads one or several .env and the corresponding env.local, env.$env and env.$env.local files if they exist.
```

Comment:
Environment variables in later files override earlier files, correct?

Reply:
Almost, see PR description.